tests/form_params_test.go

```diff
@@ -158,33 +218,39 @@ func TestFormParametersBreakUrlEncoding(t *testing.T) {
 	if err != nil {
 		t.Errorf("MakeRequest(): got err %v, want nil", err)
 	}
-	if !isOkReq(resp) {
-		t.Errorf("response: want %s, got %s", statusOK, string(resp))
+	if !isOKReq(resp) {
+		t.Errorf("response: want %s, got %s", statusOKReq, resp)
 	}
-
 }
 
 func TestBasicMultipartForm(t *testing.T) {
-	reqBody := "--123\r\n" + "Content-Disposition: form-data; name=\"foo\"\r\n" + "\r\n" + "bar\r\n" + "--123--\r\n"
+	reqBody := "--123\r\n" +
+		"Content-Disposition: form-data; name=\"foo\"\r\n" +
+		"\r\n" +
+		"bar\r\n" +
+		"--123--\r\n"
 	reqBodyLen := len(reqBody)
 	postReq := "POST / HTTP/1.1\r\n" + "Host: localhost:8080\r\n" +
-		"Content-Type: multipart/form-data; boundary=\"123\"\r\n" + "Content-Length: " + strconv.Itoa(reqBodyLen) + "\r\n" + "\r\n" + reqBody + "\r\n\r\n"
+		"Content-Type: multipart/form-data; boundary=\"123\"\r\n" + "Content-Length: " + strconv.Itoa(reqBodyLen) + "\r\n" +
+		"\r\n" +
+		reqBody + "\r\n" +
+		"\r\n"
 	resp, err := requesttesting.MakeRequest(context.Background(), []byte(postReq), func(req *http.Request) {
 
 		e := req.ParseMultipartForm(1024)
 		if e != nil {
-			log.Printf("ParseMultipartForm: want nil, got %v", e)
+			t.Errorf("ParseMultipartForm: want nil, got %v", e)
```

Comment:
You forgot to switch this into a `t.Fatalf`

Reply:
Resolved in [3786868](url)